config/config.go

```diff
@@ -615,45 +547,50 @@ func LoadConfig(path string, mandatory bool) (*Config, error) {
 		return nil, fmt.Errorf("can't apply default values: %w", err)
 	}
 
-	fs, err := os.Stat(path)
-	if err != nil {
-		if errors.Is(err, os.ErrNotExist) && !mandatory {
-			// config file does not exist
-			// return config with default values
-			config = &cfg
-
-			return config, nil
-		}
-
-		return nil, fmt.Errorf("can't read config file(s): %w", err)
-	}
-
-	var data []byte
-
-	if fs.IsDir() {
-		data, err = readFromDir(path, data)
+	var k = koanf.New("_")
+	if path == envKey {
```

Comment:
It's a relict of the conversion since I didn't touch the file loading at first. 😬

If we load from multiple sources the environment variables should override the files in my opinion. Do you agree?